STUDIO/org.openl.rules.webstudio/webapp/pages/modules/repository/node/versions.xhtml

```diff
@@ -18,27 +18,27 @@
 
         <f:facet name="header">
             <rich:columnGroup>
-                <rich:column>
+                <rich:column  style="min-width: 70px;">

```

Comment:
Could you extract these styles to CSS style classes please?
It's not a good practice to hardcode all styles in html.

Reply:
done